fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -373,6 +375,30 @@ public PlanFragment visitPhysicalHashJoin(PhysicalHashJoin<Plan, Plan> hashJoin,
         }
     }
 
+    @Override
+    public PlanFragment visitPhysicalNestedLoopJoin(PhysicalNestedLoopJoin<Plan, Plan> nestedLoopJoin,
+            PlanTranslatorContext context) {
+        // NOTICE: We must visit from right to left, to ensure the last fragment is root fragment
+        PlanFragment rightFragment = nestedLoopJoin.child(1).accept(this, context);
+        PlanFragment leftFragment = nestedLoopJoin.child(0).accept(this, context);
+        PlanNode leftFragmentPlanRoot = leftFragment.getPlanRoot();
+        PlanNode rightFragmentPlanRoot = rightFragment.getPlanRoot();
+        JoinType joinType = nestedLoopJoin.getJoinType();
+
+        if (joinType.equals(JoinType.CROSS_JOIN)
```

Comment:
Please refer to the review comments on rule predicate of `LogicalJoinToNestedLoopJoin`.